remediation/fence-agents-remediation/fence-agents-remediation.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: default
+title: Fence Agents Remediation
+nav_order: 4
+has_children: true
+parent: Remediation
+---
+
+## Fence Agents Remediation
+
+Generally Available
+{: .label .label-green }
+
+The [Fence Agents Remediation](https://github.com/medik8s/fence-agents-remediation#readme) (FAR) is a Kubernetes operator that uses external tools to *fence* unhealthy nodes.
+The tools are a set of well-known [upstream fence agents](https://github.com/ClusterLabs/fence-agents), where each one of them can be used for different environments,
+to fence a node using a traditional Application Programming Interface (API) call that reboots a node.
+By doing FAR can minimizes downtime for stateful applications and restores compute capacity if transient failures occur.
+
+FAR is not only fencing a node when it becomes unhealthy, it also tries to *remediate* the node from being unhealthy to healthy again.
+It adds a taint to evict stateless pods, fence the node with a fence agent, and fully remediate it with resource deletion to remove any remaining workloads (mostly stateful workloads) after reboot.
+
+FAR is recommended when a node becomes unhealthy, and we want to completely isolate the node from a cluster,
+since we can’t “trust” the unhealthy node, to prevent it from accessing the shared resources like [RWO volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes).
+The operator is available in the Kubernetes community, [OperatorHub.io](https://operatorhub.io/operator/fence-agents-remediation), and can be installed manually as mentioned in [FAR documentation](https://github.com/medik8s/fence-agents-remediation#installation)
+
+Similar to the other [remediation operators from Medik8s](https://www.medik8s.io/remediation/remediation/#implementations),
+FAR can semi-automate the node remediation once the node has been detected/selected to be remediated.
```

Comment:
- why semi-automate? It's fully automated, not? 🙂 
- FAR can automate -> FAR can be used to automate

Reply:
>why semi-automate? It's fully automated, not? 

Yes, I think the term `automation` doesn't fit here.